<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramUI</class>
 <widget class="QMainWindow" name="ProgramUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>767</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1111</width>
    <height>767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>USBIPManager - Main window</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QGridLayout" name="grid_layout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="menu_box">
      <property name="title">
       <string>Main menu</string>
      </property>
      <layout class="QGridLayout" name="menu_box_layout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="main_menu_layout">
         <item>
          <layout class="QVBoxLayout" name="auto_find_layout">
           <item>
            <widget class="QPushButton" name="auto_find_button">
             <property name="minimumSize">
              <size>
               <width>88</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../icon/auto_find.png</normaloff>../icon/auto_find.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="auto_find_label">
             <property name="text">
              <string>Auto find</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="add_server_layout">
           <item>
            <widget class="QPushButton" name="add_server_button">
             <property name="minimumSize">
              <size>
               <width>88</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../icon/add_server.png</normaloff>../icon/add_server.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="add_server_label">
             <property name="text">
              <string>Add server</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="search_all_layout">
           <item>
            <widget class="QPushButton" name="search_all_button">
             <property name="minimumSize">
              <size>
               <width>88</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../icon/search_all.png</normaloff>../icon/search_all.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="search_all_label">
             <property name="text">
              <string>Search all</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="connect_all_layout">
           <item>
            <widget class="QPushButton" name="connect_all_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>88</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../icon/connect_all.png</normaloff>../icon/connect_all.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="connect_all_label">
             <property name="text">
              <string>Connect all</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="disconnect_all_layout">
           <item>
            <widget class="QPushButton" name="disconnect_all_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>88</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../icon/disconnect_all.png</normaloff>../icon/disconnect_all.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="disconnect_all_label">
             <property name="text">
              <string>Disconnect all</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="settings_layout">
           <item>
            <widget class="QPushButton" name="settings_button">
             <property name="minimumSize">
              <size>
               <width>88</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../icon/settings.png</normaloff>../icon/settings.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="settings_label">
             <property name="text">
              <string>Settings</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="activity_box">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>112</height>
       </size>
      </property>
      <property name="title">
       <string>Network activity</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="server_box">
      <property name="minimumSize">
       <size>
        <width>592</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>592</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Server list</string>
      </property>
      <layout class="QGridLayout" name="server_box_layout">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scroll_area">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scroll_content">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>553</width>
            <height>389</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="scroll_content_layout"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="device_box">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="title">
       <string>Connected devices</string>
      </property>
      <layout class="QGridLayout" name="device_box_layout">
       <item row="0" column="0">
        <widget class="QTreeWidget" name="device_tree">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>140</number>
         </attribute>
         <column>
          <property name="text">
           <string>Server / Device</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>To</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>From</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QTextBrowser" name="log">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>181</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
